requirements.txt

```diff
@@ -47,21 +51,21 @@ pycparser==2.14
 pycrypto==2.6.1
 pyOpenSSL==0.15.1
 pyrsistent==0.11.9
-python-cinderclient==1.1.1
+python-cinderclient==1.6.0
```

Comment:
1.6.1 is the official client as of the Mitaka release and the recommended one to use. I forget where we fixed the paging, but at least for other reasons I would recommend going to 1.6.1.